cargo-nextest/src/dispatch.rs

```diff
@@ -412,6 +434,30 @@ impl TestBuildFilter {
                 .map_err(|err| ExpectedError::CreateTestListError { err })?,
         )
     }
+
+    fn merge_test_binary_args(&mut self, test_binary_args: &mut Vec<String>) {
+        let mut ignore_filter = Vec::new();
+        test_binary_args.retain(|s| {
+            if s == "--include-ignored" {
+                ignore_filter.push(RunIgnored::All);
+                false
+            } else if s == "--ignored" {
+                ignore_filter.push(RunIgnored::IgnoredOnly);
+                false
+            } else if !s.starts_with('-') {
+                self.filter.push(s.to_owned());
+                false
+            } else {
+                true
+            }
```

Comment:
Is this how test binaries interpret conflicts between `--include-ignored` and `--ignored`? Please double-check.

Also, `ignore_filter` can just be an `Option` I think if the last one between `--include-ignored` and `--ignored` wins.

Reply:
Verified that the test binary will return `error: the options --include-ignored and --ignored are mutually exclusive`. Will keep it consistent.